### PR TITLE
OBSDOCS-780 - Logging alerts correction

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.12+

 
Issue: [OBSDOCS-780](https://issues.redhat.com//browse/OBSDOCS-780)

Link to docs preview: https://74755--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging_alerts/default-logging-alerts.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by libander** [comment]
QE: @anpingli / @QiaolingTang / @kabirbhartiRH - Review, please. 
SME: @periklis - Review, please.

File: modules/logging-collector-alerts.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * logging/logging_alerts/default-logging-alerts.adoc
+
+:_content-type: REFERENCE
+[id="logging-collector-alerts_{context}"]
+= Logging collector alerts
+
+In logging 5.8 and later versions, the following alerts are generated by the {clo}. You can view these alerts in the {product-title} web console.
```
Comment (dfitzmau) [inline modules/logging-collector-alerts.adoc:9]: "logging" her is used a verb when it shoul, I think, be {clo}? Or maybe the "Logging 5.8 implementation"?

File: modules/logging-collector-alerts.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+|===
+| Alert Name | Message | Description | Severity
+
+| CollectorNodeDown
+| Prometheus could not scrape `namespace`/`pod` collector component for more than 10m.
+| Collector cannot be scraped.
+| Critical
+
+| CollectorHighErrorRate
+| `value`% of records have resulted in an error by `namespace`/`pod` collector component.
```
Comment (dfitzmau) [inline modules/logging-collector-alerts.adoc:21]:
```suggestion
| The `<value>`% of records have resulted in an error by `namespace`/`pod` collector component.
```

Comment (dfitzmau) [reply modules/logging-collector-alerts.adoc:21]: `namespace`/`pod  > the slash is confusing. I'm unsure if it's part of the naming convention, used as an or statement, or stating "per"?

File: modules/logging-collector-alerts.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+| `namespace`/`pod` collector component errors are high.
+| Critical
+
+| CollectorVeryHighErrorRate
+| `value`% of records have resulted in an error by `namespace`/`pod` collector component.
```
Comment (dfitzmau) [inline modules/logging-collector-alerts.adoc:26]: Maybe:

The value in percentage of records...

File: modules/logging-collector-alerts.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+= Logging collector alerts
+
+In logging 5.8 and later versions, the following alerts are generated by the {clo}. You can view these alerts in the {product-title} web console.
+
+[cols="4", options="header"]
+|===
+| Alert Name | Message | Description | Severity
+
+| CollectorNodeDown
+| Prometheus could not scrape `namespace`/`pod` collector component for more than 10m.
```
Comment (dfitzmau) [inline modules/logging-collector-alerts.adoc:16]:
```suggestion
| Prometheus could not scrape `namespace`/`pod` collector component for more than 10 min.
```

File: modules/logging-collector-alerts.adoc (hunk 3 above)
Comment (dfitzmau) [inline modules/logging-collector-alerts.adoc:26]:
```suggestion
| `value`% of records have resulted in an error by the `namespace`/`pod` collector component.
```

**Comment by dfitzmau** [comment]
Hi @libander . My review is finished and I aded comments inline.

**Comment by libander** [comment]
> Hi @libander . My review is finished and I aded comments inline.

The comments added are all on the content of the `message` field for alerts. These are the verbatim messages the user sees for these alerts. I tend to agree with all of your suggestions, but changing the alert message wording is not the scope of this PR. :)

File: modules/logging-collector-alerts.adoc (hunk 1 above)
Comment (libander) [reply modules/logging-collector-alerts.adoc:9]: "logging" is the approved short name for the logging product, this usage is correct though I see where you're coming from.

**Pull request merged**